src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/AuthorMap/AuthorMap.xaml.cs

```diff
@@ -0,0 +1,497 @@
+﻿// Copyright 2016 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Portal;
+using Esri.ArcGISRuntime.Security;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Navigation;
+
+namespace ArcGISRuntime.Desktop.Samples.AuthorMap
+{
+    public partial class AuthorMap
+    {
+        private Map _myMap;
+
+        // Constants for OAuth-related values ...
+        // URL of the server to authenticate with
+        private const string ServerUrl = "https://www.arcgis.com/sharing/rest";
+        // TODO: Add Client ID for an app registered with the server
+        private const string AppClientId = "2Gh53JRzkPtOENQq"; 
```

Comment:
Member (or global variables) should start with an underscore (_). 
Ex: _AppClientId